frontend/src/APIClients/AuthAPIClient.ts

```diff
@@ -1,8 +1,3 @@
-import {
-  FetchResult,
-  MutationFunctionOptions,
-  OperationVariables,
-} from "@apollo/client";
```

Comment:
Why was this removed? Was it an unused import?

Reply:
hmm apollo client? Is that for GraphQL?

Reply:
Yep it is lol